client/components/action-panel/README.md

```diff
@@ -14,29 +14,53 @@ import ActionPanelFooter from 'components/action-panel/footer';
 import Button from 'components/button';
 
 const ActionPanelExample = ( { translate }) => {
-  return (
-      <ActionPanel>
-        <ActionPanelBody>
-          <ActionPanelFigure inlineBodyText={ true }>
-            <img
-              src="/calypso/images/wordpress/logo-stars.svg"
-              width="170"
-              height="143"
-              alt="WordPress logo"
-            />
-          </ActionPanelFigure>
-          <ActionPanelTitle>Action panel title</ActionPanelTitle>
-          <p>
-            This is a description of the action. It gives a bit more detail and explains what we are
-            inviting the user to do.
-          </p>
-        </ActionPanelBody>
-        <ActionPanelFooter>
-          <Button className="action-panel__support-button" href="/help/contact">
-            Call to action!
-          </Button>
-        </ActionPanelFooter>
-      </ActionPanel>
-  );
+	return (
+		<div>
+			<ActionPanel>
+				<ActionPanelBody>
+					<ActionPanelFigure inlineBodyText={ true }>
+						<img
+							src="/calypso/images/wordpress/logo-stars.svg"
+							width="170"
+							height="143"
+							alt="WordPress logo"
+						/>
+					</ActionPanelFigure>
+					<ActionPanelTitle>Action panel title</ActionPanelTitle>
+					<p>
+						This is a description of the action. It gives a bit more detail and explains what we are
+						inviting the user to do.
+					</p>
+				</ActionPanelBody>
+				<ActionPanelFooter>
```

Comment:
Seems odd to have the footer both in the body and outside of it in different examples. I think we should follow the same structure.

Reply:
This is a tricky one to resolve. With the footer outside of the body and the figure on the left, the alignment of the buttons/links in the footer ends up under the image. This might be desired, but generally, where this pattern is being used the buttons are aligned with the body content.

We could change the HTML structure and/or not use floats, but we'd also need to audit where this component is being used and make sure that it's not affected. I'll do a first pass of that and see how much work it would be.

Reply:
Yeah, I can see it gets complicated, but I'd like to think that if it's inside the body of element X, then it's not a footer of element X, it's something else. It could make more sense for it to be the footer of element Y, considering that element Y is nested inside X.

Reply:
To keep things simple I've created another container component for use inside the body and changed the footer component to use it. I've called it `ActionPanelCta`, which might not be the best name, but in general, the footer is where the action links go, so it seemed appropriate.